ruby-training/trainees/alan/test_katas/02/fizzbuzz/fizzbuzz_game.rb

```diff
@@ -1,31 +1,31 @@
 class FizzBuzzGame
     
-    def self.it_is_fizz? (number)
+    def self.is_it_fizz? (number)
         number % 3 == 0
     end
 
-    def self.it_is_buzz? (number)
+    def self.is_it_buzz? (number)
         number % 5 == 0
     end
 
-    def self.it_is_fizzbuzz? (number)
-        it_is_buzz?(number) and it_is_fizz?(number)
+    def self.is_it_fizzbuzz? (number)
```

Comment:
I guess I forgot to tell you, please don't use `and` and `or` instead of `&&` and `||`. Those reserved words do the same thing but they take less precedence against other logical operators (such as `==`) so it is dangerous.